lib/matplotlib/colors.py

```diff
@@ -1215,36 +1215,57 @@ def reversed(self, name=None):
 
 class Normalize:
     """
-    A class which, when called, linearly normalizes data into the
-    ``[0.0, 1.0]`` interval.
+    A class which, when called, maps data values within the interval
+    ``[vmin, vmax]`` linearly to fit within the interval ``[0.0, 1.0]``. Data
+    with values outside ``[vmin, vmax]`` will be mapped based on *clip*.
```

Comment:
Minor wording suggestions to be more concise (data values -> values, map to fit within the interval -> map to the interval, ...)
```suggestion
    A class which, when called, maps values within the interval
    ``[vmin, vmax]`` linearly to the interval ``[0.0, 1.0]``. The mapping of
    values outside ``[vmin, vmax]`` depends on *clip*.
```